numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2236,15 +2229,15 @@ def add_newdoc(place, name, doc):
     Parameters
     ----------
     x1, x2 : array_like
-        Logical OR is applied to the elements of `x1` and `x2`.
-        They have to be of the same shape.
+        Logical OR is applied to the elements of `x1` and `x2`. $BROADCASTABLE
     $PARAMS
 
     Returns
     -------
     y : ndarray or bool
-        Boolean result with the same shape as `x1` and `x2` of the logical
-        OR operation on elements of `x1` and `x2`.
+        Boolean result of the logical OR operation applied to the elements
+        of `x1` and `x2`; the shape is determined by whether or not
+        broadcasting of one or both arrays was required.
```

Comment:
I think the phrasing could take advantage of the fact that even when the shapes match, broadcasting still happens -its jus a co-op.